plugins/default/check_mongodb_wrapper.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+######################################################################
+#
+# Nagios wrapper plugin to retrieve mongo admin password from
+# same project where nagios is located and then perform the
+# check requested
+#
+######################################################################
+
+################################
+# GLOBALS
+################################
+# Nagios exit codes
+NAGIOS_OK=0
+NAGIOS_WARN=1
+NAGIOS_CRIT=2
+NAGIOS_UNKNOWN=3
+
+################################
+# FUNCTIONS
+################################
+usage() {
+ echo "Usage:"
+ echo " -w <warn>    Warning threshold (default 0)"
+ echo " -c <crit>    Critical threshold (default 0)"
```

Comment:
Does it make sense to call

```
check_mongodb_wrapper.sh - w 42
```

?

If it doesn't, and the argument should always be 0, then we should remove the argument?